DxDispatch/src/dxdispatch/Device.cpp

```diff
@@ -15,7 +15,7 @@ static bool DebugMessageCallback(void* context, void* commandList, DWORD message
     return true;
 }
 #else
-static void DebugMessageCallback(D3D12_MESSAGE_CATEGORY cat, D3D12_MESSAGE_SEVERITY sev, D3D12_MESSAGE_ID id, LPCSTR message, void* context)
+static void __stdcall DebugMessageCallback(D3D12_MESSAGE_CATEGORY cat, D3D12_MESSAGE_SEVERITY sev, D3D12_MESSAGE_ID id, LPCSTR message, void* context)
```

Comment:
Should we use `STDAPICALLTYPE` rather than the MSVC-specific intrinsic, which might more generically work across other architectures (e.g. maybe ARM)?

Reply:
The callback signature is hard-coded to use `__stdcall` See the typedef in d3d12sdklayers.h:

```cpp
typedef void ( __stdcall *D3D12MessageFunc )( 
    D3D12_MESSAGE_CATEGORY Category,
    D3D12_MESSAGE_SEVERITY Severity,
    D3D12_MESSAGE_ID ID,
    LPCSTR pDescription,
    void *pContext);
```